src/frequenz/client/dispatch/__main__.py

```diff
@@ -574,7 +590,7 @@ async def delete(
         raise click.ClickException("Some deletions failed.")
 
 
-async def interactive_mode(url: str, key: str) -> None:
+async def interactive_mode(url: str, key: str, secret: str | None) -> None:
```

Comment:
```suggestion
async def interactive_mode(url: str, auth_key: str, sign_secret: str | None) -> None:
```

Same for the whole file.